.gitignore

```diff
@@ -47,3 +47,5 @@ nbactions.xml
 # Bitronix transaction logs
 *.btm
 
+# Folder for generated configuration properties collected by running maven build
+generated-docs
```

Comment:
The idea was to put the generated files somewhere accessible for asciidoc documentation rendering. I didn't want to keep the source packing/unpacking to get those files. In these latest changes, I've used the output property and passed it to asciidoc context so that the file is using the property rather than "guesses" where to look for generated files. This way, we can change the output location in one place. And for now, I've used the root target.